src/tests/banners/UsEoyAppealBannerTest.ts

```diff
@@ -15,34 +15,8 @@ const tickerSettings = {
 
 const isLive = true;
 
-export const UsEoyAppealNonSupportersBanner: BannerTest = {
-    name: 'UsEoyAppealNonSupporters',
-    bannerChannel: 'contributions',
-    testAudience: 'AllNonSupporters',
-    locations: ['UnitedStates'],
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    canRun: (targeting: BannerTargeting, pageTracking: BannerPageTracking) => isLive,
-    minPageViews: 2,
-    variants: [
-        {
-            name: 'control',
-            modulePath: usEoyAppeal.endpointPath,
-            moduleName: 'UsEoyAppealBanner',
-            componentType: 'ACQUISITIONS_ENGAGEMENT_BANNER',
-            tickerSettings,
-        },
-        {
-            name: 'variant',
-            modulePath: usEoyAppealWithVisual.endpointPath,
-            moduleName: 'UsEoyAppealBannerWithVisual',
-            componentType: 'ACQUISITIONS_ENGAGEMENT_BANNER',
-            tickerSettings,
-        },
-    ],
-};
-
-export const UsEoyAppealSupportersBanner: BannerTest = {
-    name: 'UsEoyAppealSupporters',
+export const UsEoyAppealBanner: BannerTest = {
+    name: 'UsEoyAppeal',
```

Comment:
shall we mention giving tuesday in the test name?

Reply:
ahhh sorry only just seen this - yeah we should! Maybe something like `USEoyAppealGT`?`